docs/vault/installation.md

```diff
@@ -15,6 +15,7 @@ At the end of this document you will have:
 - Make sure you have at least 50 GB of free disk space (ideally SSD).
 - You should have a stable Internet connection.
 - Make sure that your Vault client is running for at least 8 hours per day (you can do other things on the side).
+- Make sure [docker](https://docs.docker.com/engine/install/ ) and [docker-compose](https://docs.docker.com/compose/install/) are installed in your system. 
```

Comment:
Docker and docker-compose are only required in the quickstart options that will not be available on Kintsugi. 

I suggest adding the following after line 26:

```
### 0. Install docker and docker-compose

Make sure [docker](https://docs.docker.com/engine/install/ ) and [docker-compose](https://docs.docker.com/compose/install/) are installed in your system.

Reply:
Ok, better that way. I applied your suggestion.